ipyleaflet/leaflet.py

```diff
@@ -637,6 +638,10 @@ def compute_style(feature, colormap, choro_data):
 
         return compute_style
 
+    @default('key_on')
+    def _default_key_on(self):
```

Comment:
You actually don't need this method, I suggested doing it for the `style_callback` because it was cleaner in this special case, but for the `key_on` you can define it as `key_on = Unicode('id')`. It will then default to 'id'.

Reply:
Ohh sure I'll do the changes.